osquery/CMakeLists.txt

```diff
@@ -33,9 +33,32 @@ if(WIN32)
   ADD_OSQUERY_LINK_CORE("rpcrt4.lib")
   ADD_OSQUERY_LINK_CORE("shlwapi.lib")
   ADD_OSQUERY_LINK_CORE("wbemuuid.lib")
+  ADD_OSQUERY_LINK_CORE("linenoise.lib")
+  ADD_OSQUERY_LINK_CORE("boost_system-vc140-mt-1_59")
+  set(boost_system-mt_library ${boost_system-vc140-mt-1_59_library})
```

Comment:
This looks gross, but it's tolerable ;) Any reason why the rename matters; ideally it shouldn't?